assets/javascripts/discourse/templates/components/chat-live-pane.hbs

```diff
@@ -59,6 +59,7 @@
         isHovered=(eq message.id hoveredMessageId)
         onHoverMessage=(action "onHoverMessage")
         onSwitchChannel=onSwitchChannel
+        didResendStagedMessage=this.didResendStagedMessage
```

Comment:
Shouldn't this be called `resendStagedMessage` given that it's not a hook, as in, it is not adding a callback to the didResendStagedMessage event.

Reply:
I changed this name many times 😅 Im ok with `resendStagedMessage` will make the change